resources/lang/ep_gui.uk-UA.rc

```diff
@@ -29,8 +29,8 @@ BEGIN
     IDS_SYM_FAILEDALL_B         "Для цієї збірки не вдалося завантажити жодних символів. Це може бути пов'язано з тим, що поточна збірка ОС є надто свіжою або через нестабільне інтернет підключення."
 
-    IDS_CRASH_TITLE             "На жаль, Файловий Провідник аварійно завершив роботу :("
-    IDS_CRASH_BODY              "Здається, що файловий провідник несподівано завершує роботу %1$s менш ніж за %2$d секунд, щоразу під час запуску. Це може вказувати на проблему, спричинену ExplorerPatcher, який ймовірно застарів для нещодавніх змін у Windows, наприклад, під час запуску на новій збірці ОС.\nОсь кілька рекомендацій:\n• Якщо доступна нова версія, ви можете <A HREF=""%3$s"">оновити ExplorerPatcher й перезавантажити Файловий Провідник</A>.\n• На GitHub ви можете <A HREF=""%4$s"">переглянути releases</A>, <A HREF=""%5$s"">перевірити поточний статус</A>, <A HREF=""%6$s"">discuss</A> or <A HREF=""%7$s"">переглянути останні проблеми</A>.\n• Якщо ви підозрюєте, що це не пов'язано з ExplorerPatcher, видаліть усі нещодавно встановлені розширення Файлового Провідника чи схожі інструменти.\n• Якщо на даний момент рішення недоступне, ви можете <A HREF=""%8$s"">видалити ExplorerPatcher</A>, а потім перевстановити його, коли оновлення буде опубліковано на GitHub. Навіть після видалення, конфігурація програми буде збережена.\n\nПерепрошую через незручності, які це може спричинити; Я роблю все можливе, щоб підтримувати цю програму в робочому стані.\n\nExplorerPatcher вимкнено до наступного перезапуску Файлового Провідника, щоб ви могли переконатися що все працює, та зробити потрібні зміни."
+    IDS_CRASH_TITLE             "На жаль, Провідник аварійно завершив роботу :("
+	IDS_CRASH_BODY              "Здається, що провідник несподівано завершує роботу %1$s менш ніж за %2$d секунд, одразу після запуску. Це може бути через ExplorerPatcher, який ймовірно застарів після останніх змін у Windows, наприклад, під час запуску на новій збірці ОС.\nОсь кілька рекомендацій:\n• Якщо доступна нова версія, ви можете <A HREF=""%3$s"">оновити ExplorerPatcher й перезавантажити Провідник</A>.\n• На GitHub ви можете <A HREF=""%4$s"">переглянути нові версії</A>, <A HREF=""%5$s"">перевірити поточний статус</A>, <A HREF=""%6$s"">обговорити</A> чи <A HREF=""%7$s"">переглянути останні проблеми</A>.\n• Якщо ви впевнені, що це не пов'язано з ExplorerPatcher, видаліть усі нещодавно встановлені розширення Провідника чи схожі інструменти.\n• Якщо на даний момент рішення недоступне, ви можете <A HREF=""%8$s"">видалити ExplorerPatcher</A>, а потім перевстановити його, коли оновлення доступне на GitHub. Навіть після видалення, конфігурація програми буде збережена.\n\nПерепрошую через незручності, які це може спричинити; Я роблю все можливе, щоб підтримувати цю програму в робочому стані.\n\nExplorerPatcher буде вимкнено до наступного перезавантаження Провідника, щоб ви могли переконатися що все працює, та зробити потрібні зміни."
     IDS_CRASH_ONCE              "Один раз"

```

Comment:
Change to lowercase first letter because it gets substituted into %1$s

Reply:
I've added changes, please review if possible.
Additionally, I fixed several errors that I found.